goRename.py

```diff
@@ -285,7 +290,7 @@ def gorename(self, file_path, begin_offset=None, flags=None, name=None, callback
         # Build gorename cmd.
         cmd = "%(toolpath)s -offset %(file_path)s:%(pos)s -to %(name)s %(flags)s" % {
         "toolpath": toolpath,
-        "file_path": file_path,
+        "file_path": cmd_quote(file_path),
```

Comment:
sorry for the trouble. it would be great if the canonical path could be escaped, i'm proposing:

```python
"file_path": cmd_quote(os.path.realpath(file_path)),
```
instead of
```python
 "file_path": cmd_quote(file_path),
```